MarshallAsch/veil-droid
# 98 display notifications

**Overview:**
This PR adds in functional notifications to the application. The notification displays the post title an the author name. It also takes into account if the anonymous field is check. If so the notification won't display the author's name.

**Testing:**
- make a post and see if the notification is up
- minimize the app
- check if the notification is display in the post title and the author name
- clicking the notification should route you to the discover forums page


This closes #98
